src/page/homepage/Homepage.jsx

```diff
@@ -42,19 +43,24 @@ export const HomePage = () => {
   });
 
   return (
-    <div className="main-page">
-      <main>
-        <div className="d-flex home-head">
-          <Filter />
-          <AddNoteBtn />
-        </div>
-        <NotesInputModal />
-        <div className="notes-section">
-          {filteredList &&
-            filteredList.length &&
-            filteredList.map((item) => <NoteCard key={item._id} item={item} />)}
-        </div>
-      </main>
-    </div>
+    <>
+      <SideNav />
+      <div className="main-page">
+        <main>
+          <div className="d-flex home-head">
+            <Filter />
+            <AddNoteBtn />
+          </div>
+          <NotesInputModal />
+          <div className="notes-section">
+            {filteredList &&
+              filteredList.length &&
```

Comment:
suggestion: we can use filteredList?.length here instead of filteredList && filteredList.length